site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -168,3 +169,35 @@ It is also not possible to rename or otherwise customize scopes used with Entity
 
 * An example link:/docs/v1/tech/oauth/#example-client-credentials-grant[client credentials grant using Entities].
 * The link:/docs/v1/tech/apis/entity-management/[Entity Management APIs].
+
+== SCIM Integration
+[NOTE.since]
+====
+This functionality has been available since 1.36.0
+====
+
+// TODO: need screenshots
+
+When configuring FusionAuth to accept SCIM requests, you must create a SCIM Server Entity and a SCIM Client Entity. The Entity Types can be created by selecting the down arrow next to the add icon icon:plus[role=ui-button green,type=fas] on the link:#entity-types[Entity Type homepage]. However, by default FusionAuth ships with the two SCIM Entity Types provided. An external SCIM Client must be of the Entity Type SCIM Client and have the associated permissions enabled to successfully call authenticated link:/docs/v1/tech/apis/scim[SCIM API endpoints].
+
+==== [FusionAuth Default] SCIM Client
+The default SCIM Client Entity Type is the external SCIM client calling link:/docs/v1/tech/apis/scim[SCIM API endpoints].
+
+==== [FusionAuth Default] SCIM Server
+The default SCIM Server Entity Type is FusionAuth acting as the SCIM server and defines the available permissions for calling link:/docs/v1/tech/apis/scim[SCIM API endpoints].
+
+==== [FusionAuth Default] SCIM Server Permissions
```

Comment:
```suggestion
==== SCIM Server Permissions
```

These aren't default, correct? If I wanted to create a new SCIM Entity Type, I'd have to use these same permissions.

Reply:
Well, they are created as part of the default scim server entity type. Wasn't sure of the best way to document that.

Reply:
Yes, that is correct. I see your point. I'll remove the default text.